Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/Storage/Merchant.yaml

```
@@ -1011,6 +1011,7 @@ TransporterConfig:
     generateReferralCodeForOperator: Maybe Bool
     maxAllowedDocSizeInMB: Maybe Int
     bulkWaiveOffLimit : Int
+    rcExpiryChecks : Maybe Bool
```

YAMLlint (1.35.1)

[warning] too many spaces before colon

(colons)